src/backend/wsl.ts

```diff
@@ -150,7 +150,7 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
   protected progressInterval: ReturnType<typeof timers.setInterval> | undefined;
 
   /** The version of Kubernetes currently running. */
-  protected activeVersion: semver.SemVer | null = null;
+  protected activeVersion: semver.SemVer | undefined;
```

Comment:
It might be nice to use the `?` syntax so that the `| undefined` can be omitted. But I'm guessing you wrote it this way on purpose - I'm easy either way.

Reply:
I did it this way because I thought `?` was only applicable to function signatures (it's not); see also line 150 (visible in the context here).

Ideally, we'd pick one and have an eslint rule for it, but I didn't spot anything applicable.

Reply:
Yeah it's fine. I think `?` is easier to miss, so less readable, at least for properties.